playground/nextjs-app-router/components/AppProvider.tsx

```diff
@@ -119,6 +122,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     '0x1D6b183bD47F914F9f1d3208EDCF8BefD7F84E63:1',
   );
 
+  const [isSponsored, setIsSponsoredState] = useState<boolean>(false);
```

Comment:
I think we can omit State like so:

```suggestion
  const [isSponsored, setIsSponsored] = useState<boolean>(false);
```